apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderContext.cs

```diff
@@ -35,7 +35,7 @@ internal sealed class TraceHeaderContext
             new TraceHeaderContext(null, null, null);
 
         /// <summary>The trace header.</summary>
-        internal const string TraceHeader = "X-Cloud-Trace-Context";
+        public const string TraceHeader = "X-Cloud-Trace-Context";
```

Comment:
How certain are we that this really won't change in the future? It might be worth turning into a readonly property.